### DISCUSSION TITLE
Possible to selectively enable directory downloads?

### DISCUSSION THREAD

**Discussion by studiomav** [DirectoryLister/DirectoryLister#883] [Help & Support] [answered]
I'm planning on hosting a large repository of files, sorted into many folders. I _don't_ want users to be able to "download this directory" from the root index page, as that would queue a potentially resource intensive zip download of, well, everything. Is it possible to configure something like a .ignoredownload file that will be hidden, but also disable a directory it's present in from being downloaded?

**Comment by PHLAK** [comment, marked as answer]
This is not currently possible but definitely something we'll consider adding in the future.
